tests/test_variance_threshold_select.py

```diff
@@ -0,0 +1,38 @@
+from feature_selection import variance_threshold_select
+import numpy as np
+import seaborn as sns
+import pandas as pd
+import pytest
+
+iris = sns.load_dataset('iris')
```

Comment:
Can we load the iris dataset from [sklearn](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_iris.html) instead? This way we don't need to include seaborn.